earth_enterprise/BUILD_RPMS.md

```diff
@@ -47,13 +47,20 @@ allowed Open GEE Fusion and Server to be separately installed when building out
 of the Git repo checkout directly.
 
 ## Installing RPMs
-After the SCons **package_install** target completes you can install and
-test the RPMs you have created on CentOS or RHEL.  To do this, change to the
-*rpms/build/distributions* directory.  You can then use ```rpm -Uhv
-opengee-postgis-*.rpm``` and ```rpm -Uhv opengee-common-*.rpm``` to get the
-base dependencies installed, in that order.  You can then install ```rpm -Uhv
-opengee-fusion-*.rpm``` and/or ```rpm -Uhv opengee-server-*.rpm``` as desired,
-and then you can test the RPM packaged Open GEE Fusion and Server.
+After the SCons **package_install** target completes you can install and test
+the RPMs you have created on CentOS or RHEL.  To do this, change to the
```

Comment:
I'm not sure if it makes much difference, but I usually use `sudo yum localinstall *.rpm` instead. I don't know if there is any particular benefit to using yum, but it might be more straight forward.